qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -13,9 +13,9 @@ class MgrModuleTestCase(DashboardTestCase):
     MGRS_REQUIRED = 1
 
     @classmethod
-    def tearDownClass(cls):
-        cls._ceph_cmd(['mgr', 'module', 'disable', 'telemetry'])
-        super(MgrModuleTestCase, cls).tearDownClass()
+    def setUpClass(cls):
+        cls._ceph_cmd(['telemetry', 'off'])
```

Comment:
Is this really required? IMO this code can be removed.

Reply:
Please add a comment like you did in the PR description.

Reply:
> Is this really required? IMO this code can be removed.

I'll put a comment in the code to make it clear this:
2) Triggered 'ceph telemetry off' to avoid 'TELEMETRY_CHANGED' health checks warning
that was causing test timeouts.

Reply:
> Is this really required? IMO this code can be removed.

After stopping my old cluster and run again the tests in a fresh new cluster,
I found out that you were right and this is not required, so I removed the code and the comment in the PR.